ci-operator/step-registry/openshift/microshift/infra/sos/openshift-microshift-infra-sos-ref.yaml

```diff
@@ -1,24 +1,16 @@
 ref:
   as: openshift-microshift-infra-sos
-  from: dependency-payload
+  from: microshift-test-payload
```

Comment:
We dont really need anything from the dependency payload image here. Any image with a shell will do.